alibi_detect/cd/classifier.py

```diff
@@ -43,6 +44,7 @@ def __init__(
             device: Optional[str] = None,
             dataset: Optional[Callable] = None,
             dataloader: Optional[Callable] = None,
+            input_shape: Optional[tuple] = None,
```

Comment:
`input_shape` added to allow saving of tensorflow models (it was missing from some detectors which aren't currently save-able).